File: Leetcode/Daily/20-05-2024/prob.cpp
class Solution {
public:
    int subsetXORSum(vector<int>& nums) {
        int sum = 0;
        int len = nums.size();
        for(int i=1;i<(1<<len);i++){
            int x= 0;
            for(int j=0;j<len;j++){
                if((i>>j)&1){
                    x ^= nums[j];
                }
            }
            sum += x;
        }

        return sum;
    }
};

File: Leetcode/Daily/20-05-2024/prob.go
func subsetXORSum(nums []int) int {

    sum := 0
    l := len(nums)
    for i:=1;i<(1<<l);i++{
        x := 0
        for j:=0;j<len(nums);j++{
            if (i>>j)&1 == 1 {
                x = (x^nums[j])
            }
        }
        sum += x
    }

    return sum
}